ert3/engine/_record.py

```diff
@@ -23,5 +23,5 @@ def sample_record(
 ) -> ert.data.RecordCollection:
     distribution = parameters_config[parameter_group_name].as_distribution()
     return ert.data.RecordCollection(
-        records=[distribution.sample() for _ in range(ensemble_size)]
+        records=tuple(distribution.sample() for _ in range(ensemble_size))
```

Comment:
I guess this is a change in that we now only allow for tuples? I don't really mind as I'm a big fan of tuples anyway...

Reply:
Yes, pydantic did implicit conversion, and that's something we could do, I just opted not to.

Reply:
I don't mind? However, there seems to be some logic for `sets` etc right? Or am I misreading the new implementation of records?

Reply:
Yes, that is the one I was referring to, but I see now that that is for processing the numerical record data. While what is now passed as tuples instead are the list of records. I support these changes! Thanks for clarifying :)